--- arch/arm/boot/dts/Makefile	2016-01-31 21:29:37.000000000 +0200
+++ arch/arm/boot/dts/Makefile.new	2016-07-27 17:57:49.216344143 +0200
@@ -43,6 +43,13 @@
 	at91-kizbox2.dtb \
 	at91-sama5d2_xplained.dtb \
 	at91-sama5d3_xplained.dtb \
+	kses-bp-ek.dtb \
+	kses-bp-lite.dtb \
+	kses-rsm-blue.dtb \
+	kses-glam-comms.dtb \
+	kses-glam-comms-static.dtb \
+	kses-rsm-utility.dtb \
+	kses-rsm-blue3g.dtb \
 	sama5d31ek.dtb \
 	sama5d33ek.dtb \
 	sama5d34ek.dtb \
