--- drivers/net/ethernet/cadence/macb.c	2016-01-31 21:29:37.000000000 +0200
+++ drivers/net/ethernet/cadence/macb-new.c	2016-02-29 12:45:55.669287665 +0200
@@ -2758,7 +2758,7 @@
 };
 
 static const struct macb_config sama5d3_config = {
-	.caps = MACB_CAPS_SG_DISABLED | MACB_CAPS_GIGABIT_MODE_AVAILABLE,
+	.caps = MACB_CAPS_SG_DISABLED | MACB_CAPS_GIGABIT_MODE_AVAILABLE | MACB_CAPS_USRIO_DEFAULT_IS_MII,
 	.dma_burst_length = 16,
 	.clk_init = macb_clk_init,
 	.init = macb_init,
