--- drivers/net/phy/Kconfig	2016-01-31 21:29:37.000000000 +0200
+++ drivers/net/phy/Kconfig.new	2016-02-08 09:14:28.782393428 +0200
@@ -115,6 +115,12 @@
 	tristate "Drivers for ICPlus PHYs"
 	---help---
 	  Currently supports the IP175C and IP1001 PHYs.
+	  
+config IP175D_PHY
+    tristate "Driver for ICplus IP175D PHY"
+    ---help---
+      Configures IP175D for VLAN functionality.
+	  
 
 config REALTEK_PHY
 	tristate "Drivers for Realtek PHYs"
--- drivers/net/phy/Makefile	2016-01-31 21:29:37.000000000 +0200
+++ drivers/net/phy/Makefile.new	2016-02-08 09:14:51.505478663 +0200
@@ -42,3 +42,5 @@
 obj-$(CONFIG_MDIO_BCM_UNIMAC)	+= mdio-bcm-unimac.o
 obj-$(CONFIG_MICROCHIP_PHY)	+= microchip.o
 obj-$(CONFIG_MDIO_BCM_IPROC)	+= mdio-bcm-iproc.o
+obj-$(CONFIG_IP175D_PHY)    += ip175d.o
+
